site/js/index.js

```
@@ -1,223 +1,223 @@
+    return false;
   }
-  return true
-}
+  return true;
+};
 
-const CURR_SCRIPT_VERSION = 16;
+const CURR_SCRIPT_VERSION = 17;
 const INFO_DEFAULT = '{"hash":"clem"}';
 
 function updateData(fromVersion, toVersion) {
```

ESLint

[error] 'toVersion' is defined but never used.

(no-unused-vars)